modules/Dockerfile

```
@@ -3,17 +3,17 @@ FROM ${NGINX_FROM_IMAGE} as builder
 
 ARG ENABLED_MODULES
 
-RUN set -ex \
-    && if [ "$ENABLED_MODULES" = "" ]; then \
+SHELL ["/bin/bash", "-exo", "pipefail", "-c"]
```

Wouldnt this affect all further invocations of a resulting image?  Maybe we should just do it like we did with "set -ex", and add pipefail there for the scripts we explicitely chosen to have it?

---

What negative effect are you thinking of?
In general you don't want to have a image build where a pipe silently fails.
And if you add it only on demand, then you have to remember this when you maybe add later a pipe command.

Many people use this in Dockerfiles and shell scripts for years (including me), without experiencing any problem with this.
It is also the recommended by hadolint: https://github.com/hadolint/hadolint/wiki/DL4006